core-java-modules/core-java-string-algorithms/src/test/java/com/baeldung/reverse/ReverseStringExamplesUnitTest.java

```diff
@@ -67,4 +67,37 @@ public void whenReverseTheOrderOfWordsUsingApacheCommonsIsCalled_ThenCorrectStri
         assertEquals(StringUtils.EMPTY, reversedEmpty);
     }
 
+    @Test
+    public void whenReverseStringUsingIntStreamRangeMethod_ThenCorrectStringIsReturned() {
+        String reversed = ReverseStringExamples.reverseUsingIntStreamRangeMethod(STRING_INPUT);
+        String reversedNull = ReverseStringExamples.reverseUsingIntStreamRangeMethod(null);
+        String reversedEmpty = ReverseStringExamples.reverseUsingIntStreamRangeMethod(StringUtils.EMPTY);
+
+        assertEquals(STRING_INPUT_REVERSED, reversed);
+        assertEquals(null, reversedNull);
```

Comment:
We can simplify this kind of asserts using: assertNull(reversedNull);

Reply:
Done